zypp/target/rpm/RpmDb.cc

```diff
@@ -1400,65 +1401,37 @@ bool RpmDb::systemReadLine( std::string & line )
   {
     process->setBlocking( false );
     FILE * inputfile = process->inputFile();
-    int    inputfileFd = ::fileno( inputfile );
-    do
-    {
-      /* Watch inputFile to see when it has input. */
-      fd_set rfds;
-      FD_ZERO( &rfds );
-      FD_SET( inputfileFd, &rfds );
-
-      /* Wait up to 5 seconds. */
-      struct timeval tv;
-      tv.tv_sec = 5;
-      tv.tv_usec = 0;
+    do {
+      // check every 5 seconds if the process is still running
```

Comment:
Please extend the comment and mention that this is done in case a rpm script launches a deamon which does not properly close it's stdin/out/err. In this case the input pipe remains  open after rpm itself has finished.

Reply:
Done